### PR TITLE
[#25] Setup localization

### PR DESCRIPTION
**Pull request by luongvo**
https://github.com/luongvo/flutter-survey/issues/25

## What happened 👀

We need to support multi-languages due to can increase our user base.
 
## Insight 📝

- Follow the guide from https://flutter.dev/docs/development/accessibility-and-localization/internationalization to setup i18n.
- Add support to have `en` and `vi` translations. 
 
## Proof Of Work 📹

- EN

- VI

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -15,6 +17,16 @@ class MyApp extends StatelessWidget {
...
       home: LoginPage(),
+      localizationsDelegates: [
+        AppLocalizations.delegate,
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        GlobalCupertinoLocalizations.delegate,
+      ],
+      supportedLocales: [
+        Locale('en', ''),
+        Locale('vi', ''),
```
Comment (doannimble) [inline lib/main.dart:28]: To avoid hardcoding support locales, I suggest to use : 
AppLocalizations.delegate.supportedLocales instead

**Review by markgravity — APPROVED** [review]
LGTM

File: lib/main.dart (hunk 1 above)
Comment (sleepylee) [reply lib/main.dart:28]: that's good to know 🙌

Comment (luongvo) [reply lib/main.dart]: @sleepylee resolved 👌

**Review by sleepylee — APPROVED** [review]

**Pull request merged**